packages/cw22/src/query.rs

```diff
@@ -0,0 +1,8 @@
+use cosmwasm_schema::cw_serde;
+
+#[cw_serde]
+pub struct VersionResponse {
```

Comment:
Let's remove this. I don't define smart queries for this, they require more wiring.

Let's be like cw2 (and cw4) and focus on spec'ing the storage layout for efficient queries. The contract should be able to easily write its own storage. Other contracts will be querying it's storage (Map.query) and testing the resulting value.